src/CameraControls.ts

```diff
@@ -500,6 +500,21 @@ export class CameraControls extends EventDispatcher {
 		const lastDragPosition = new THREE.Vector2() as _THREE.Vector2;
 		const dollyStart = new THREE.Vector2() as _THREE.Vector2;
 
+		const disposePointer = ( pointer?: PointerInput ) => {
```

Comment:
Can `pointer` be a required argument?
I would prefer like:
```ts
const pointer = ...
pointer && disposePointer( pointer );
```

Reply:
Done

Reply:
Of course - done 😁